source/val/instruction.h

```diff
@@ -66,6 +66,10 @@ class Instruction {
   /// The words used to define the Instruction
   const std::vector<uint32_t>& words() const { return words_; }
 
+  const spv_parsed_operand_t& operand(size_t idx) const {
```

Comment:
Comment please.

Reply:
Also, size_t is excessive since by SPIR-V format there are at most 64K words and therefore at most 64K operands (though I don't know of a case of more than 15 or so).

But I see you have changed some loops to use size_t indices.   So I'm not that concerned about this.

Reply:
The size_t matches the existing GetOperandAs method as well. Using size_t here keeps things consistent.